kubejs/server_scripts/constants/general.js

```diff
@@ -0,0 +1,61 @@
+//priority: 1001
+
+const air = 'minecraft:air';
```

Comment:
rename file to `air.js` and only keep this line? :D

Reply:
✅